[HW] Introduction to html and css

Alexander-Khamitski opened the pull request:
[Introduction to html and css] task 1 and task 2

Commit:
[Introduction to html and css] task 1

Commit:
[Introduction to html and css] task 2

DHrytsuk commented on src/main/webapp/WEB-INF/hw/task1/chemicalElements.html:
```diff
@@ -0,0 +1,85 @@
+        </h1>
+        <hr>
+        <table>
+            <tr>
```
Let's add `thead`

Alexander-Khamitski replied:
Ok, it's not a problem. I did it like `<tr>` because according the task column headers don't look bold: 
<img width="419" alt="image" src="https://github.com/Alexander-Khamitski/TMS_Java_EE/assets/93436427/1dd9d63e-22e4-4b66-b08b-76cc32fda136">

Alexander-Khamitski replied:
@DHrytsuk Updated: [7cc84ffc](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/1/commits/7cc84ffc8f5d0a8d90cbb644b232f3835fddc3fb)

DHrytsuk commented on src/main/webapp/WEB-INF/hw/task1/chemicalElements.html:
```diff
@@ -0,0 +1,85 @@
+                <td>
+                    <div width="30px">
+                        <a href="https://ru.wikipedia.org/wiki/%D0%92%D0%BE%D0%B4%D0%BE%D1%80%D0%BE%D0%B4">
+                            <img src="/resources/article_image.png" alt="article_image" style="width:30px; height:30px" class="center">
```
Do we need `class="center"`?

Alexander-Khamitski replied:
@DHrytsuk Updated: [7cc84ffc](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/1/commits/7cc84ffc8f5d0a8d90cbb644b232f3835fddc3fb)

DHrytsuk commented on src/main/webapp/WEB-INF/hw/task1/chemicalElements.html:
```diff
@@ -0,0 +1,85 @@
+                <td>
+                    <div>
+                        <a href="https://ru.wikipedia.org/wiki/%D0%93%D0%B5%D0%BB%D0%B8%D0%B9">
+                            <img src="/resources/article_image.png" alt="article_image" style="width:30px; height:30px">
```
If you move some common stiles to head, let's do the same for `img`

Alexander-Khamitski replied:
@DHrytsuk Updated: [7cc84ffc](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/1/commits/7cc84ffc8f5d0a8d90cbb644b232f3835fddc3fb)

DHrytsuk commented on src/main/webapp/WEB-INF/hw/task2/quotes.html:
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="/src/main/webapp/WEB-INF/hw/task2/quotesStyle.css">
```
We could use relative path

Alexander-Khamitski replied:
Early `resources` package was placed in a root of my project. So, I moved package to `task1` directory and now relative path is: `../resources/article_image.png`

Alexander-Khamitski replied:
@DHrytsuk Updated: [7cc84ffc](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/1/commits/7cc84ffc8f5d0a8d90cbb644b232f3835fddc3fb)

Commit:
[Introduction to html and css] updated img path to relative
moved img style to head block
updated columns headers to <thead>
deleted useless class="center" from img
